integration-tests/testconfig/vrfv2/vrfv2.toml

```diff
@@ -48,110 +51,68 @@ bhs_job_lookback_blocks = 250
 bhs_job_poll_period = "1s"
 bhs_job_run_timeout = "24h"
 
-# load test specific config
-[Load.VRFv2]
-[Load.VRFv2.Common]
-cancel_subs_after_test_run = true
 
-[Load.VRFv2.General]
-minimum_confirmations = 3
-randomness_request_count_per_request = 3 # amount of randomness requests to make per one TX request
-randomness_request_count_per_request_deviation = 2 #NOTE - deviation should be less than randomness_request_count_per_request setting
-number_of_sub_to_create = 1
+# PERFORMANCE test specific config
 
-[Load.VRFv2.Performance]
-# approx 60 RPM - 1 tx request with 3 rand requests in each tx every 3 seconds
-rate_limit_unit_duration = "3s"
-rps = 1
-
-[Load.VRFv2.NewEnv]
-sub_funds_link = 1000
-node_sending_key_funding = 1000
-
-[Load.VRFv2.ExistingEnv]
+[VRFv2.ExistingEnv]
+coordinator_address = "0x9Ff8b0fe6563288d6b43ca705268BD99DfD4d2bF"
+consumer_address = ""
 sub_id = 1
+key_hash = "0x904d5d3a5fd0b02acc61c7a16773cbb9342e30979766972ef9e90fc6e11b4415"
```

Comment:
I understand that this is not a secret value and can be committed?

Reply:
exactly